independent-publisher-connectors/Zoom Meetings/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Zoom Client ID",
+          "scopes": [
+            "meeting:read:admin meeting:read"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://zoom.us/oauth/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://zoom.us/oauth/token"
+            },
+            "refreshUrl": {
+              "value": "https://zoom.us/oauth/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Akuthota Deekshith",
+    "stackOwner": "Zoom"

```

Comment:
@A-Deekshith This service/API is owned by `"Zoom Video Communications, Inc."`, I am not sure why you have provided just "Zoom".

Reply:
@sriyen-msft I had made the changes suggested by you.